spec/models/task_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+RSpec.describe Task do
+  describe 'associations' do
+    it { is_expected.to belong_to(:project) }
+  end
+
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_length_of(:name).is_at_least(3).is_at_most(60) }
```

Comment:
3 and 60 is magic numbers. Please, use constant which should be defined in model. (why previous comment resolved?)